mmcv/ops/nms.py

```diff
@@ -302,3 +303,53 @@ def nms_match(dets, iou_threshold):
         return [dets.new_tensor(m, dtype=torch.long) for m in matched]
     else:
         return [np.array(m, dtype=np.int) for m in matched]
+
+
+def nms_rotated(boxes, scores, iou_threshold, labels=None, multi_label=False):
+    """Performs non-maximum suppression (NMS) on the rotated boxes according to
+    their intersection-over-union (IoU).
+
+    Rotated NMS iteratively removes lower scoring rotated boxes which have an
+    IoU greater than iou_threshold with another (higher scoring) rotated box.
+
+    Args:
+        boxes (Tensor):  Rotated boxes in shape (N, 5).
+        iou_threshold (float): IoU thresh for NMS.
+        labels (Tensor): boxes's label in shape (N,).
+        multi_label (bool): use multi label nms rotated.
+
+    Returns:
+        tuple: kept dets(boxes and scores) and indice, which is always the \
+            same data type as the input.
+    """
+    if boxes.shape[0] == 0:
+        return boxes
+    if multi_label:
+        assert labels is not None
+    dets = boxes[:, :5]
+    scores = boxes[:, 5]
```

Comment:
There is a bug here. The score is passed as an independent argument now, and boxes should be of shape (N, 5).